tfhub_dev/assets/sayakpaul/mobilenetv2-coco/1.md

```diff
@@ -0,0 +1,9 @@
+# Placeholder sayakpaul/deeplabv3-mobilenetv2/1
+Lightweight deep learning model for semantic image segmentation.
+
+<!-- module-type: image-segmentation -->
+<!-- network-architecture: DeepLab (mobilenetv2_coco_voc_trainval) -->
+<!-- dataset: PASCAL VOC 2012 -->
+<!-- fine-tunable: false -->
+<!-- language: en -->
```

Comment:
The language metadata property is not applicable to image models, so we can remove this (unless I'm missing something).

Reply:
@maringeo fixed it.